### PR TITLE
updated htmls for Kodebeagle

### PR DESCRIPTION
**Pull request by uidev547**

### CODE REVIEW

**Comment by ScrapCodes** [comment]
There is a small type in the line `Results showin for: FileChannel` Change it to `Results shown for: FileChannel`

**Comment by ScrapCodes** [comment]
Can you remove : `More info` page for now ?

**Comment by ScrapCodes** [comment]
@hashd Please take a look !

**Comment by hashd** [comment]
@uidev547 Would it be possible to refactor the code into separate files for development. Do maintain a build process for concatenation and minification to build one big file which can be used in the webpage.

**Comment by uidev547** [comment]
@hashd: I am happy to do the change. But this change will not go in this merge request.

**Comment by ScrapCodes** [comment]
image

Please take a look at the search window.

File: library/dev/js/ng-modules/demo.js [hunk 1]
```diff
@@ -1,356 +1,363 @@
...
+      } ]
+    )
+    .directive('initModel', [
+      'model',
+      '$location',
+      '$rootScope',
+      'docsService',
+      '$document',
+      function(model, $location, $rootScope, docsService, $document) {
+        var esURL = 'http://192.168.2.145:9201';
```
Comment (ScrapCodes) [inline library/dev/js/ng-modules/demo.js]: Wrong ES URL. I will fix this myself.

File: demov1.html [hunk 2]
```diff
@@ -1,363 +0,0 @@
...
-
-<script src="//cdnjs.cloudflare.com/ajax/libs/angular-ui-bootstrap/0.13.0/ui-bootstrap.min.js"></script>
-<script src="//cdnjs.cloudflare.com/ajax/libs/angular-ui-bootstrap/0.13.0/ui-bootstrap-tpls.js"></script>
-<script src="library/dev/js/vendors/ui-ace.js" type="text/javascript"></script>
-<script src="library/dev/js/factories/http-service.js"></script>
-<script src="library/dev/js/ng-modules/demo.js"></script>
-<script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
```
Comment (ScrapCodes) [inline demov1.html]: you have deleted my GA script. :( I will restore it.

File: index.html [hunk 3]
```diff
@@ -10,136 +9,178 @@
...
+	            (i[r].q = i[r].q || []).push(arguments)
+	        }, i[r].l = 1 * new Date();
+	        a = s.createElement(o),
+	                m = s.getElementsByTagName(o)[0];
+	        a.async = 1;
+	        a.src = g;
+	        m.parentNode.insertBefore(a, m)
+	    })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
+
+	    ga('create', 'UA-60790831-1', 'auto');
```
Comment (ScrapCodes) [inline index.html]: This is the wrong GA script.

**Comment by ScrapCodes** [comment]
I have merged this with my own changes on top of it.

**Pull request closed without merging**